src/utils/functions.ts

```diff
@@ -0,0 +1,7 @@
+export function getDiscountedPrice(price: number, discount: number): number {
+  return price - (price * discount);
+}
+
```

Comment:
Esta tambien 


```suggestion
/**
 * This function bla, bla, bla
 * @param {number} price - Original price
 * @param {number} discount - Discount 
 * @return {number} - Discounted price
 */
export function getDiscountedPrice(price: number, discount: number): number {
  return price - (price * discount);
}
```

Also, podrías hacer discount un parametro opcional?

Reply:
es que esa función solo se ejecuta si el producto tiene discount, eso lo tengo en product card por eso no lo puse como parámetro opcional